src/rgw/driver/d4n/rgw_sal_d4n.cc

```diff
@@ -361,7 +361,7 @@ int D4NFilterObject::D4NFilterReadOp::prepare(optional_yield y, const DoutPrefix
 }
 
 int D4NFilterObject::D4NFilterDeleteOp::delete_obj(const DoutPrefixProvider* dpp,
-					   optional_yield y)
```

Comment:
I know there's a habit in RGW code of adding boolean arguments, but especially in interfaces we don't want to constantly adjust, we might consider a flag word and named constants for the bits?

Reply:
@mattbenjamin can you please take a look at this interface change when you have a chance?

Reply:
thank you, looks great